src/screens/transfer/amount/useAmountLogic.tsx

```diff
@@ -38,37 +38,35 @@ export function useAmountLogic({ prevPage, nextPage, mainButtonText, onAmountCha
     touched && !transferAll && maxAmountToSend && amount && +maxAmountToSend - +amount < deposit,
   );
 
-  const handleAmountFee = async (selectedAsset: TrasferAsset, transferAmmount: string) => {
-    const fee = isStatemineAsset(selectedAsset.asset.type)
-      ? await getAssetHubFee(
-          selectedAsset.chainId,
-          selectedAsset.asset.typeExtras!.assetId,
-          transferAmmount,
-          selectedAsset.address,
-          selectedAsset?.isGift,
```

Comment:
We still need to understand whether it is a gift or not
or
To have it in a more similar way to the rest of the tokens, we can calculate here only one fee and add +1 fee in `sendGift` in `useExtrinsic`. It was hard to do with the current implementation with ED AH but without it - should be fine